### PR TITLE
DNF2: Argparser2

### PR DESCRIPTION
**Pull request by MichaelMraka**
updated core plugins to new argparser
depends on rpm-software-management/dnf#474

### CODE REVIEW

**Comment by jsilhan** [comment]
can you remove `--repo` param from reposync.rst, please? Otherwise looks good to me.

**Comment by jsilhan** [comment]
+ (RhBug:1260986)

**Comment by j-mracek** [comment]
Jenkins, test this please

File: plugins/repoquery.py [hunk 1]
```diff
@@ -269,10 +258,6 @@ def installonly(self, q):
             return installonly
 
     def run(self, args):
-        if self.opts.help_cmd:
-            print(self.parser.format_help())
-            return
-
```
Comment (mluscon) [inline plugins/repoquery.py]: Is there any way how to output a full help for plugin?

Comment (MichaelMraka) [inline plugins/repoquery.py]: from plugin:
```
self.base.optparser.print_help(self)
```

Comment (mluscon) [inline plugins/repoquery.py]: I meant from command line.

**Comment by mluscon** [comment]
merged

**Pull request closed without merging**